app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -346,7 +346,9 @@
               Primaire contactgegevens 
             </:title>
             <:infotext>
-               De primaire contactgegevens kunnen niet langer in het OP aangepast worden. De organisaties kunnen deze gegevens zelf aanpassen via de Contactapp.
```

Comment:
**Comment:** Imo the resulting page looks a bit weird for an organisation without an address. If the organisation has no address yet the data card shows an empty box underneath the message:
![2024-06-03-184013_995x114_scrot](https://github.com/lblod/frontend-organization-portal/assets/95910348/a83953d5-58f7-4a5b-944d-fffea46ecea5)

Could we make in a bit more obvious that an address has just not been added yet?

Reply:
I agree, I will deploy and see what Sofi think about